python/python/lance/dataset.py

```diff
@@ -2298,8 +2299,34 @@ def optimize_indices(self, **kwargs):
         the new data to existing partitions.  This means an update is much quicker
         than retraining the entire index but may have less accuracy (especially
         if the new data exhibits new patterns, concepts, or trends)
-        """
-        self._dataset._ds.optimize_indices(**kwargs)
+
+        Parameters
+        ----------
+        merge_indices: bool | int | List[str]
+            If True, all indices will be merged. If False, no indices will be
+            merged and instead a new index delta will be created. If an integer,
+            that number of indices will be merged. If a list of UUID strings,
+            those specific indices will be merged.
```

Comment:
I got a little confused here because I didn't know if you were talking about "pass in UUIDs if you only want to update some columns (e.g. multiple vector embeddings, each with an index, and only update a few)" or if you were talking about "pass in UUIDs here if you have multiple deltas in the same column to merge together"